use-cases/run_jcl/scripts/cleanup_output.js

```diff
@@ -0,0 +1,52 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the *
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at  *
+ * https://www.eclipse.org/legal/epl-v20.html                                            *
+ *                                                                                       *
+ * SPDX-License-Identifier: EPL-2.0                                                      *
+ *                                                                                       *
+ * Copyright Contributors to the Zowe Project.                                           *
+ */
+
+/******************************************************************** 
+ *   This script is intended to be executed from the command line   *
+ *   at your terminal or in a Jenkins (or other CI/CD tool)         * 
+ *   pipeline.                                                      *
+ *                                                                  *
+ *   The script will remove the output/ directory.                  *
+ ********************************************************************/
+const path = require("path");
+const fs = require("fs");
+
+try {
+    const sourceDir = path.join(__dirname, "..", "output");
+    console.log("Removing 'output' directory...");
+    rmDirR(sourceDir);
+    console.log("+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~+")
+    console.log("| 'output' directory deleted! |");
+    console.log("+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~+")
+    console.log("");
+} catch (err) {
+    console.error("ERROR!!! Unable to remove source directory 'output'. Please delete manually!");
+    console.error(`Error Details: ${err.message}`);
+    console.error("");
+    process.exit(1);
+}
+
+/**
+ * Recurisvely delete a directory and its contents.
+ * @param {*} path The directory to delete.
+ */
+function rmDirR(path) {
```

Comment:
Curious why this function is defined instead of using [`rimraf`](https://www.npmjs.com/package/rimraf)? Do we want to avoid extra dependencies? Sorry I'm probably missing something 😢

Reply:
I kinda just took this script from the sonarqube_datasets example from the CLI example scripts repository, and recreated with APIs in this repository. We could use rimraf if we want to.